mama/c_cpp/src/c/bridge/avis/publisher.c

```diff
@@ -100,7 +101,9 @@ avisBridgeMamaPublisherImpl_prepareMessage (mamaMsg* msg)
         return ret;
     }
 
-    if (MAMA_PAYLOAD_AVIS == payload)
+    // TODO: Should replace this with a more run-time way to get the payload ID
```

Comment:
Should we spin out an issue for covering this in some later release? Just so we don't commit a TODO and never address it.